webapp/src/main/java/gov/nist/healthcare/ttt/webapp/common/controller/TempUploadController.java

```diff
@@ -22,20 +23,29 @@
 
 @Controller
 @RequestMapping("/api/upload")
+
 public class TempUploadController {
 	
+	public static boolean exceededFileSizeLimit;
```

Comment:
This should be private
`private static boolean exceededFileSizeLimit`
Hence why you have created a public getter, otherwise that wouldn't be needed and you'd have the less secure option of using direct dot notation, e.g, TempUploadController.exceededFileSizeLimit vs exceededFileSizeLimit.isValidFileSize()